tests/integration/api_server/test_adapter.py

```diff
@@ -106,6 +106,22 @@ async def test_api_server_json(host):
     await asyncio.gather(*tasks)
 
 
+@pytest.mark.asyncio
+async def test_api_server_json_with_customized_route(host):
```

Comment:
I am getting errors when I run pytest on api_server.  429 'Too Many Requests'

Reply:
It's normal. It happens when your test server is busy.